README.md

```diff
@@ -12,5 +12,15 @@ FPGA
 # Configuration and file locations
 
 In `/etc/netdata/netdata.conf` in the `plugins` section the following line should be present: `python.d = yes`  
-The configuration file `fpga.conf` should be placed in `/etc/netdata/conf.d/python.d/`  
-The python file `fpga.chart.py` should be placed in `/usr/libexec/netdata/python.d/`  
+
+In order to install the Swarm64 FPGA monitoring scripts for Netdata, run the `install.py` script included in
+this repository. It will do some basic checks and then attempt to install the configuration script and the
+chart script either in the default directories or optionally to other locations specified by the user.
+Please run the script with `--help` to see this usage.
+The script will not overwrite existing Swarm64 scripts that happen to be in the locations specified unless the
+`--force` option is specified. This is also detailed with the running of `--help`.
+
+Once the scripts are successfully installed please restart Netdata in order to see the FPGA statistics displayed.
+The charts need the Swarm64da database to be available in order to get the statistics. The chart script attempts
```

Comment:
"Swarm64 DA extension in PostgreSQL loaded" (instead of: Swarm64da database to be available)